### PR TITLE
Improve servey 20210917

### LINKED ISSUE #521: Help & About menu always be white color
**Describe the bug**
Help & About menu always white color

**To Reproduce**
Steps to reproduce the bug:
1. Go to 'Other'
2. Click on 'Night Mode'
3. Select  'Dark'
4. See bug

**Expected behavior**
Help & About menu should be dark mode

**Log**

**Screenshots**

**Smartphone (please complete the following information):**
 - Device: [Redmi]
 - OS: [Android 11]
 - Version: [3.2.2]

**Additional context**

### PR DESCRIPTION
**Pull request by WhiredPlanck**
#### Issue tracker
Fixes will automatically close the related issue

Fixes #521，#528，#494

#### Manual testing
- [x] Done

#### Build tasks success
Successfully running following tasks on local
- [x] [CONTRIBUTING](CONTRIBUTING.md)
- [x] `./gradlew spotlessCheck`
- [x] `./gradlew assembleDebug`

#### Code Reviews
1. No wildcards import
2. Manual build and test pass
3. GitHub action ci pass
4. At least one contributor reviews and votes
5. Can be merged clean without conflicts
6. PR will be merged by rebase upstream base

#### Daily build for review
Fetch artifact after login

https://github.com/osfans/trime/actions

### CODE REVIEW

File: app/src/main/java/com/osfans/trime/ime/core/Trime.java [hunk 1]
```diff
@@ -150,6 +146,83 @@ private Preferences getPrefs() {
...
+              case BOTTOM_LEFT:
+              default:
+                x = 0;
+                y = yRight;
+                break;
+            }
+          } else {
+            // setCandidatesViewShown(false);
+            if (winPos == WindowsPositionType.RIGHT || winPos == WindowsPositionType.RIGHT_UP) {
+              // 此处存在bug，暂未梳理原有算法的问题，单纯根据真机横屏显示长候选词超出屏幕进行了修复
```
Comment (ameaninglessname) [inline app/src/main/java/com/osfans/trime/ime/core/Trime.java:198]: 是不是可以加个TODO 方便后续跟进

Comment (WhiredPlanck) [reply app/src/main/java/com/osfans/trime/ime/core/Trime.java:198]: 我觉得这样的表面修复目前够用，整理完这里后测试感觉没什么问题，暂时不用再动。

Comment (ameaninglessname) [reply app/src/main/java/com/osfans/trime/ime/core/Trime.java:198]: 噢，我的意思是在注释里
`// TODO 或者 todo 或者 @todo`
 方便IDE识别到这个，就是备忘而已

我不会安卓开发，哈哈

Comment (WhiredPlanck) [reply app/src/main/java/com/osfans/trime/ime/core/Trime.java:198]: 没事，我们都记得住这个地方 ～

**Comment by WhiredPlanck** [comment]
Thanks!

**Pull request merged**